src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summay.spec.ts

```diff
@@ -2,10 +2,10 @@ import { ComponentFixture, async, TestBed } from '@angular/core/testing';
 import { DrivingFaultSummaryComponent } from '../driving-fault-summary';
```

Comment:
Could you do a quick rename of this file please, there's a typo in the filename.

Reply:
took me a while to see the spelling mistake but renamed after I did.